### ISSUE TITLE
Consuming txt-Files via upload

### ISSUE THREAD

**Issue by andbez** [jonaswinkler/paperless-ng#338]
Txt-Files can be uploaded to paperless-ng via Webfrontend and will be consumed.
1. This Files won't be converted to pdfs
2. If you would like to edit (in paperless) the document, it is not shown in preview
3. Show Document works.

**Comment by jonaswinkler** [comment]
1. That's true, I couldn't find an easy way to convert text documents into PDF documents yet. `enscript` has some issues with UTF-8 encoded files.
2. Editing files isn't supported by paperless at all right now. I'll add support for showing text documents in the preview, But that'll really just show the same text that's also available in the content field right now.

**Comment by andbez** [comment]
Sorry, did not mean "editing the document", but the edit-view in paperless with the preview-frame of the document. The Preview-Frame is empty. Please see attached Screenshot.
<img width="1013" alt="Bildschirmfoto 2021-01-14 um 13 58 35" src="https://user-images.githubusercontent.com/63553946/104593901-aafd8780-5670-11eb-8811-5fe9c66a6dbf.png">

**Comment by jonaswinkler** [comment]
Ah, okay. For some reason I though you want to edit the actual .txt document with paperless.

Preview for all file types will be there, eventually.

**Comment by jovandeginste** [comment]
Gotenberg supports `txt` files; we could add it there. Alternatively, we could create a txt2pdf converter using what's already present.

**Comment by jonaswinkler** [comment]
This is a basic feature and I don't want to use an optional service to provide that.

**Comment by jovandeginste** [comment]
Would you find it reasonable to install `a2ps` inside the image? (It's available in the default repos).

A test with a `LICENSE.txt` I had laying around, gave this:

[LICENSE.txt](https://github.com/jonaswinkler/paperless-ng/files/5826356/LICENSE.txt)

```bash
$ a2ps LICENSE.txt -R -=book --header="Generated by Paperless-ng" -o LICENSE.ps
[LICENSE.txt (plain): 3 pages on 3 sheets]
[Total: 3 pages on 3 sheets] saved into the file `LICENSE.ps'
$ ps2pdf LICENSE.ps LICENSE.pdf
```
Result:

[LICENSE.pdf](https://github.com/jonaswinkler/paperless-ng/files/5826355/LICENSE.pdf)

(NB: `ps2pdf` is already installed)

**Comment by jonaswinkler** [comment]
Sure, either that or `enscript`.

I'm just having issues with different file encodings when special characters are used (äöüßèéê etc)

**Comment by jovandeginste** [comment]
You're right; I now tried this tool, and it seems to work: [txt2pdf](https://github.com/baruchel/txt2pdf)

It's also written in Python.
